content/docs/command-reference/version.md

```diff
@@ -113,26 +109,23 @@ Inside a DVC project:
 ```dvc
 $ dvc version
 
-DVC version: 0.41.3+f36162
-Python version: 3.7.1
-Platform: Linux-4.15.0-50-generic-x86_64-with-debian-buster-sid
-Binary: False
-Cache: reflink - False, hardlink - True, symlink - True
-Supported remotes: azure, gdrive, gs, hdfs, http, https, s3, ssh, oss
-Filesystem type (cache directory): ('ext4', '/dev/sdb3')
+DVC version: 1.7.4 (pip)
+---------------------------------
+Platform: Python 3.8.3 on Linux-5.4.0-47-generic-x86_64-with-debian-buster-sid
+Supports: All remotes
+Cache types: hardlink, symlink
```

Comment:
Is this your actual output atm? You don't get `reflink` on Linux by default?

Reply:
No, I didn't get `reflink` . I think this is because of `ext4` file system.
According to this [blog](https://dev.to/robogeek/reflinks-vs-symlinks-vs-hard-links-and-how-they-can-help-machine-learning-projects-1cj4), `reflink` works only on `BTRFS`, `XFS` and `OCFS2` file systems.

Reply:
Good find. Can you quickly check if this is documented somewhere other than the blog? How did you find that blog post BTW, Google? 🙂

Reply:
Yeah I googled for information on `reflinks`.

Reply:
OK. Can you quickly check if this is documented somewhere other than the blog?

Reply:
Other than this only source I was this [wikipedia](https://en.wikipedia.org/wiki/Comparison_of_file_systems) page about filesystems. In table Allocation and layout policies, there is a column named "copy on write" (which I found to be same as reflinks). It shows which all filesystems support "copy on write".

Reply:
No, I meant in our docs of course. Is this information or some general note that reflinks only work on some file systems in there? Not in the blog:

> According to this blog, reflink works only on BTRFS, XFS and OCFS2 file systems

Reply:
It's actually document in the docs,[here](https://dvc.org/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) in file link types.

Reply:
So [File link types](https://dvc.org/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache) shows up on google while searching "reflinks supported file systems" but it comes on like 3th or 4th page.

Reply:
OK. So we're covered then. DVC is not about "reflinks supported file systems" at all so I'm surprised we even come up in search results. BTW not sure why we're talking about SEO..? Anyway, resolving!